super_csv/serializers.py

```diff
@@ -3,26 +3,42 @@
 """
 from __future__ import absolute_import, unicode_literals
 
+import json
+
 from django.contrib.auth import get_user_model
 from rest_framework import serializers
 
 from .models import CSVOperation
 
 
+class CSVOperationDataSerializer(serializers.Serializer):  # pylint: disable=abstract-method
+    """
+    Serializer for data in CSV bulk operation summary results info
+    """
+    total_rows = serializers.IntegerField()
+    processed_rows = serializers.IntegerField()
+    saved_rows = serializers.IntegerField()
+
+
 class CSVOperationSerializer(serializers.ModelSerializer):
     """
-    Serializer for history CSV bulk grade operations
+    Serializer for CSV bulk operations
     """
 
     user = serializers.SlugRelatedField(
         slug_field='username',
         queryset=get_user_model().objects.all()
     )
+    data = serializers.SerializerMethodField()
 
     class Meta:
         model = CSVOperation
-        fields = ('class_name', 'unique_id', 'operation', 'user', 'modified', 'original_filename')
+        fields = ('id', 'class_name', 'unique_id', 'operation', 'user', 'modified', 'original_filename', 'data')
 
     @classmethod
     def get_related_queryset(cls, queryset):
         return queryset.select_related('user')
+
+    def get_data(self, operation):
+        data = json.loads(operation.data.read())
```

Comment:
For future reference, you can use `json.load(operation.data)` -- loading json from a file vs from a string can potentially be more efficient for big objects.

Reply:
Shall I make this change elsewhere in super-csv as well? https://github.com/edx/super-csv/blob/master/super_csv/mixins.py#L117

Reply:
Moving forward assuming yes. If it's safe to do here, it's probably safe to do elsewhere